File: frontend/src/app/stats/stats.ts
export interface Stats {
    accidentsPerYear: {
        year: number;
        nbAccident: number;
    }[];
    averageBusLifeTime: number;
    averageReparationTime: number;
    furnitureCostPerYear: {
        year: number;
        furnitureCostPerYear: number;
    }[];
    averageSeniorityDriver: number;
}

File: backend/src/main/java/fr/iut/valence/but/info/ctvr/backend/dto/Stats.java
package fr.iut.valence.but.info.ctvr.backend.dto;

import java.util.List;

public record Stats(
        List<StatsAccidentPerYear> accidentsPerYear,
        Double averageBusLifeTime,
        Double averageReparationTime,
        List<StatsFurnitureCostPerYear> furnitureCostPerYear,
        Double averageSeniorityDriver
) {
}
